src/tests/system/tests/test_logging.py

```diff
@@ -100,64 +158,107 @@ def test_logging__default_settings_logs_domain_configuration_errors(client: Clie
+        1. Check default domain status
+        2. Check domain log
+    :expectedresults:
+        1. SSSD is not connected
+        2. Logs contain connection/timout errors
+    :customerscenario: False
+    """
+    client.sssd.stop()
+    client.firewall.outbound.reject_host(ldap)
+    client.sssd.domain["ldap_network_timeout"] = "1"
+    client.sssd.enable_responder("ifp")
+    client.sssd.start(debug_level=None, raise_on_error=False)
+
+    time.sleep(60)
+
+    result = client.sssctl.domain_status(client.sssd.default_domain if client.sssd.default_domain else "test")
+    assert result is not None, "Could not get domain status!"
+    assert "Online status: Offline" in result.stdout, "LDAP is connected!"
+
+    logs = client.fs.read(client.sssd.logs.domain())
+    assert "Failed to connect, going offline" in logs, "Offline error messages are not in logs!"
+
+    assert (
+        "No available servers for service 'LDAP'" in logs
+    ), "'No available servers for service' error message is not in logs!"
+    assert (
+        "Connection timed out [ldap_network_timeout]" in logs
+    ), "'Connection timed out [ldap_network_timeout]' error message is not in logs!"
+
+
+@pytest.mark.integration
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_logging__default_settings_logs_to_syslog_offline_errors(client: Client, ldap: LDAP):
     """
-    :title: Log to syslog when sssd cannot contact ldap servers and the servers go offline
+    :title: Log to syslog when sssd can not contact (ldap) servers and goes offline
     :setup:
-        1. Configure SSSD with an invalid uri and start SSSD
+        1. Configure SSSD with ifp responder and LDAP backend
+        2. Reduce ldap_network_timeout and ldap_opt_timeout to 1 second
+        3. Block LDAP access using firewall
+        4. Start SSSD with default debug level and wait a bit for sssd to start
     :steps:
         1. Check domain status using sssctl
-        2. Clear syslog and restart SSSD and check syslog
+        2. Check syslog/journal
     :expectedresults:
         1. Domain is offline
-        2. Logs contain SSSD errors
-    :customerscenario: True
+        2. Syslog/journal contains SSSD offline error
+    :customerscenario: False
     """
-    client.sssd.domain["ldap_uri"] = "ldaps://typo.invalid"
-    client.sssd.start()
-
-    assert client.sssd.default_domain is not None, "Failed to load default domain!"
-    status = client.sssctl.domain_status(client.sssd.default_domain)
-    assert status is not None
-    assert "Offline" in status.stdout or "Unable to get online status" in status.stderr, "Domain is not offline!"
 
+    client.sssd.stop()
     client.journald.clear()
-    client.sssd.restart()
-    time.sleep(1)
+    client.firewall.outbound.reject_host(ldap)
+    client.sssd.domain["ldap_opt_timeout"] = "1"
+    client.sssd.domain["ldap_network_timeout"] = "1"
+    client.sssd.start(debug_level=None, raise_on_error=False)
+    time.sleep(95)
```

Comment:
95 seconds! I know overhead and time are important to you, but why does this value have to be so high?

Reply:
(1) Can this test be merged with previous test?
(2) How exactly does it fail with low timeout?

Reply:
Merged them to together and added polling loops instead of fixed sleeps.